server/schedule/checker_controller.go

```diff
@@ -70,11 +73,26 @@ func (c *CheckerController) CheckRegion(region *core.RegionInfo) []*operator.Ope
 
 	if c.opts.IsPlacementRulesEnabled() {
 		if op := c.ruleChecker.Check(region); op != nil {
+			added := false
+			// ignore spilt or merge
```

Comment:
When will it have the merge operator?

Reply:
I think the regions will be nil if the operator is merge,so check miss peers will remove this region from queue